articles/consuming-apis-in-flutter/index.md

```diff
@@ -0,0 +1,250 @@
+Application Programming Interface (API) is a communication portal that allows two or more applications to connect for data sharing. It acts as an intermediary for delivering requests to service providers and returning the responses. The use of APIs has gained prevalence in mobile application development, given the ease of using pre-existing frameworks. Programmers use most APIs to fetch data from web servers and render it to the application's UI components.
+
+This article will demonstrate how to fetch and consume data from a server using a RESTful API in a flutter application. You can download the code for this application form [here](https://github.com/jerimkaura/flutter-book/tree/main/mygithub).
```

Comment:
The GitHub link specified here returns a 404.

![image](https://user-images.githubusercontent.com/34317373/112730676-e4981c00-8f58-11eb-8604-28b192197089.png)